### PR TITLE
suppress e2e logs

### PR DESCRIPTION
**Pull request by zdeveloper**
# BACKEND PULL REQUEST

## Related Issue

- e2e are logging way to much info, that makes it harder to load the logs cause it freezes
 
## Changes Proposed

- Suppress verbose e2e logs
  - disable hibernate query logs
  - disable db logs

### CODE REVIEW

**Comment by emmastephenson** [comment]
From eng sync: turning off these docker logs very consistently causes e2es to fail on Okta connections.

A proposed alternative is to disable logging at the backend and database level, rather than through Docker (in `application-e2e` and the database start script)

File: docker-compose.cypress.yml [hunk 1]
```diff
@@ -57,6 +57,10 @@ services:
...
     hostname: localhost.simplereport.gov
+    environment:
+      NGINX_ENTRYPOINT_QUIET_LOGS: 1
   mailhog:
     profiles:
-      - donotstart
\ No newline at end of file
+      - donotstart
+  db:
+    command: -p ${SR_DB_PORT:-5432} -c shared_buffers=512MB -c max_connections=500
```
Comment (zdeveloper) [inline docker-compose.cypress.yml:66]: removes query logging

**Review by BobanL — APPROVED** [review]
Logs still look useful for troublshooting and are significantly smaller (11k to now 4k‼️)

**Review by mehansen — APPROVED** [review]
thanks for doing this
excited to be able to load and view cypress logs!

**Pull request merged**